src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
it looks cleaner that way
```suggestion
        Lottery ball1 = new Lottery();
        Lottery ball2 = new Lottery();
        Lottery ball3 = new Lottery();
        System.out.println(ball1.getRandomBall().toString());
        System.out.println(ball2.getRandomBall().toString());
        System.out.println(ball3.getRandomBall().toString());
```